storybook/src/stories/Button.stories.svelte

```diff
@@ -0,0 +1,68 @@
+<script context="module" lang="ts">
+import { faBell } from '@fortawesome/free-regular-svg-icons';
+import { Button } from '@podman-desktop/ui-svelte';
+import { Story, Template } from '@storybook/addon-svelte-csf';
+
+export const meta = {
+  title: 'Example/Button',
```

Comment:
should it contain 'Example' ?

Reply:
Yes this is for the nav bar path, they will be under Example/